### PR TITLE
Feat: add eval cmd, util, and travis

### PR DESCRIPTION
**Pull request by KurokuTetsuya**
Yaudah gitu aja. yaudah. review tinggal review njng

### CODE REVIEW

**Comment by KurokuTetsuya** [comment]
@dwicao oke thank youuuuuu <3

File: .gitignore [hunk 1]
```diff
@@ -72,3 +72,5 @@ typings/
 .config 
 
 .glitch-assets
+
+template.js
```
Comment (dwicao) [inline .gitignore]: what this is for?

Comment (KurokuTetsuya) [reply .gitignore]: tadinya buat ignore si template buat eslint tapi gajadi

Comment (dwicao) [reply .gitignore]: remove it lah

Comment (KurokuTetsuya) [reply .gitignore]: ah alright, thanks.

File: src/commands/developers/EvalCommand.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      aliases: ["ev", "e"],
+      devOnly: true,
+      guildOnly: true,
+      nsfw: false,
+      clientPermission: [],
+      authorPermission: []
+    });
+  }
+
+  async execute(client, message, args, thread) {
```
Comment (dwicao) [inline src/commands/developers/EvalCommand.js]: this code should can ONLY run by authors

Comment (KurokuTetsuya) [reply src/commands/developers/EvalCommand.js]: ofc, scroll confignya

Comment (KurokuTetsuya) [reply src/commands/developers/EvalCommand.js]: udah ada handlernya pak, tinggal set `devOnly: true` otomatis jadi owner only

Comment (dwicao) [reply src/commands/developers/EvalCommand.js]: oh my bad, I'm not seen the `devOnly: true`
ok then

**Comment by KurokuTetsuya** [comment]
okay, thanks for approving my request.

**Comment by KurokuTetsuya** [comment]
Alright, I'll merge this branch into master immediately.

**Pull request merged**